tfhub_dev/assets/silero/models/silero-stt/en/1.md

```diff
@@ -0,0 +1,76 @@
+# Module silero/silero-stt/en/1
+A set of compact enterprise-grade pre-trained STT Models for multiple languages.
+
+<!-- asset-path: https://silero-models.ams3.cdn.digitaloceanspaces.com/models/en/en_v1_tf.tar -->
```

Comment:
Is it possible to compress (`gzip`) the `en_v1_tf.tar` asset to `en_v1_tf.tar.gz`?

Reply:
Yes, ofc, I will do it later today

Reply:
@maringeo 
I re-uploaded the tgz file and changed the link in the PR